test/compress/issue-3768.js

```diff
@@ -66,3 +66,63 @@ compress: {
     }
     expect_stdout: "number undefined boolean string undefined"
 }
+
+call_arg_1: {
+    mangle = {
+        toplevel: true,
+    }
+    input: {
+        var z = "foo";
+        (function() {
+            var z = false;
+            (function(e) {
+                var z = 42;
+                e("console.log(typeof z)");
+            })(eval);
+        })();
+    }
+    expect: {
+        var z = "foo";
+        (function() {
+            var o = false;
+            (function(o) {
+                var a = 42;
+                o("console.log(typeof z)");
+            })(eval);
+        })();
+    }
+    expect_stdout: "undefined"
```

Comment:
Can we also use `expect_stdout: true` here?
It produces "string" in the browser.

Reply:
Might as well &minus; GitHub Actions just failed to even do `git checkout` 😅